frontend/src/metabase/lib/formatting/date.tsx

```diff
@@ -127,13 +127,16 @@ export function formatDateTimeForParameter(value: string, unit: DatetimeUnit) {
 
 /** This formats a time with unit as a date range */
 export function formatDateTimeRangeWithUnit(
-  value: string | number,
+  value: string | number | (string | number)[],
```

Comment:
```suggestion
  value: string | number | [string | number, string | number],
```
Using a 2-tuple here makes the type-checking more specific, that way an array with more or less than two values cannot be provided as the argument.

Reply:
ae30c685dd38d911404f6cd18f78444fdab2f8ee